.github/workflows/ci-compress-images.yml

```diff
@@ -0,0 +1,29 @@
+name: Compress Images
+on:
+  workflow_dispatch:
+  # schedule:
+  #   - cron: '00 23 * * 0'
```

Comment:
noting here that (if I read it correctly) this means every Sunday at 11 pm

Reply:
I believe that is correct, yes